internal/overlord/servstate/handlers.go

```diff
@@ -601,6 +612,31 @@ func (s *serviceData) stop() error {
 	return nil
 }
 
+func (s *serviceData) waitProcessGroup() error {
+	var (
+		status syscall.WaitStatus
+		rusage syscall.Rusage
+		pid1   int
+		err    error
+	)
+	for {
+		// __WALL (since Linux 2.4)
+		// Wait for all children, regardless of type ("clone" or "non-clone").
+		pid1, err = syscall.Wait4(-s.cmd.Process.Pid, &status, syscall.WALL, &rusage)
+		// https://linux.die.net/man/2/waitid
+		// wait(): on success, returns the process ID of the terminated child; on error, -1 is returned.
+		if pid1 == -1 {
+			break
+		}
+		logger.Noticef("Waited for child %d of PGID %d", pid1, s.cmd.Process.Pid)
```

Comment:
Should this be `Debugf`? We probably don't want to know about it in the normal case.

Reply:
Yes, changed to Debugf (this is what I was using it for in the first place) :)